### DISCUSSION TITLE
How to set a Search in a Preset?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11450] [Archive]
It looks like the Search is just looking there for a word in any field of the items of the Collection.

Creating `My Collection` with `Status` and one input `Title`

and the 4 items
| Title   | Status  |
|---|---|
| Title 1   |  draft |
| Title 2   |  published |
| Title 3   |  archived |
| Title 4 not yet archived   | published  |

If you set the filter to `archived`, you will see the 2 items:
| Title   | Status  |
|---|---|
| Title 3   |  archived |
| Title 4 not yet archived   | published  |
<br>
    
Is it possible to set the search on the `Status` field in order to have only?
| Title   | Status  |
|---|---|
| Title 3   |  archived |

**Comment by rijkvanzanten** [comment]
This can't currently be configured from the presets page. A workaround is to create a bookmark from the regular content module first, and then upgrade that to a global bookmark/preset from settings afterwards 👍🏻 

(we're currently working on an update to have the full filter/search experience available on the preset detail page as well)
